src/routes/(public)/functions/deploy/+page.svelte

```diff
@@ -46,6 +63,15 @@
             queries: [Query.equal('teamId', selectedOrg), Query.orderDesc('')]
         });
 
+        // Fetch plan info for billing checks
+        if (isCloud && selectedOrg) {
+            try {
+                currentPlan = await sdk.forConsole.billing.getOrganizationPlan(selectedOrg);
```

Comment:
there's a `plansInfo` derived store that has cached plan details, we could use that to avoid a request. If a plan isn't found in there, we should fallback to making the API call.